pyproject.toml

```diff
@@ -15,7 +15,7 @@ nbconvert = ">=6.0.7"
 nbformat = "5.1.3"
 nh3 = ">=0.2.17"
 requests = ">=2.25.1"
-rpy2 = "3.4.4"
+rpy2 = ">=3.4.4"
```

Comment:
stopped pinning this version because there were dependency issues with the build